cmd/user-agent-js-worker/src/worker-loader-web.js

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+export function loadWorker(pending, notifications, paths) {
+    const wasmJS = paths.wasmJS
+    const wasm = paths.wasm
+    const workerJS = paths.dir + "/worker-impl-web.js"
+    const worker = new Worker(workerJS + "?wasmJS=" + wasmJS + "&wasm=" + wasm)
+    worker.onmessage = e => {
+        const result = e.data
+        if (result.topic){
```

Comment:
we don't need this check if we are not using sending topics from wasm. (notifier mechanism).

Reply:
I need it for https://github.com/trustbloc/edge-agent/pull/76/files#diff-62d1e052170f4151cbcabd61f36085e6R291